build/DependencyVersions.props

```diff
@@ -13,7 +13,7 @@
     <NuGetVersion>4.5.0-preview2-4529</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25908-02</NETStandardLibraryNETFrameworkVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</NETStandardLibraryNETFrameworkVersion>
```

Comment:
I'm curious what the diff is here that fixes the issue. Do you have a pointer to the change?

Reply:
Here is the PR https://github.com/dotnet/corefx/pull/25786. Basically what we did is to add a netfx.force.conflicts.dll file to the net471 closure so that we force the generation of binding redirects for the 12 shims we are servicing for net 4.7.1